modules/aws-outposts-ingress-controller-service.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-ingress-controller-service_{context}"]
+= Configuring the default Ingress Controller in an AWS VPC cluster extended into an Outpost
+
+Classic Load Balancers in the AWS VPC cluster can attach to edge compute resources in the Outpost if edge and cloud-based subnets are in the same availability zone.
+As a result, Classic Load Balancers on the VPC cluster might schedule pods on either of these node types.
+
+The default Ingress Controller deployment on the VPC cluster is scheduled by a Classic Load Balancer.
+Because the load balancer can attach to edge compute resources in the Outpost, it might schedule Ingress Controller deployment on edge compute nodes.
+Scheduling on these nodes introduces latency between the router and the backend.
+To prevent the default Ingress Controller in the VPC cluster from attaching to Outpost edge compute nodes, you must apply labels to the cloud-based compute nodes and configure the default Ingress Controller to only schedule on nodes with the applied labels.
```

Comment:
```suggestion
To prevent the default Ingress Controller in the VPC cluster from scheduling on Outpost edge compute nodes, you must apply labels to the cloud-based compute nodes and configure the default Ingress Controller to only schedule on nodes with the applied labels.
```